src/Network/WebSockets/TSSocketManager.m

```diff
@@ -381,15 +379,22 @@ - (void)processWebSocketRequestMessage:(WebSocketRequestMessage *)message {
 
         if (!decryptedPayload) {
             DDLogWarn(@"Failed to decrypt incoming payload or bad HMAC");
```

Comment:
I think we should display a user facing error here, it's arguably out of scope, so feel free to add a new card.